pkgs/by-name/go/goto/package.nix

```
@@ -0,0 +1,65 @@
+{
+  coreutils,
+  fetchFromGitHub,
+  gawk,
+  goto,
+  lib,
+  nix-update-script,
+  runCommand,
+  stdenvNoCC,
+}:
+
+let
+  owner = "iridakos";
+  rev = "b7fda54e0817b9cb47e22a78bd00b4571011cf58";
+
+in
+stdenvNoCC.mkDerivation {
+  pname = "goto";
+  version = "2.1.0";
```

The version of the latest commit is `2.0.0`
```suggestion
  version = "2.0.0";
```

---

There is an untagged 2.1.0 release in the last commit

---

Right it should be `2.1.0-unstable-2020-11-15` then